shell/browser/api/electron_api_menu_mac.mm

```diff
@@ -24,6 +25,24 @@
 
 static scoped_nsobject<NSMenu> applicationMenu_;
 
+ui::Accelerator GetAcceleratorFromKeyEquivalentAndModifierMask(
```

Comment:
Can we not make use of [`GetKeyEquivalentAndModifierMaskFromAccelerator`](https://source.chromium.org/chromium/chromium/src/+/master:ui/base/accelerators/platform_accelerator_cocoa.mm;l=13-41q=%22%20NSEventModifierFlagShift%22&ss=chromium) in Chromium?

Reply:
@codebytere That function is the literal inverse of this one, the naming is similar because I manually reversed the Chromium impl for doing the opposite of what I wanted 😄